CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+
+## `jupyter_starters 1.1.0f1`
```

Comment:
Ideally roll back the various version changes, as there might be other things that need updating: we have only done relatively-well-understood `alpha`/`a`  releases in the past, but usually just try to stick to well-tested major/minor/patch versions.

Reply:
(or set to `1.1.1`)